Back-End/Dockerfile

```diff
@@ -1,14 +1,27 @@
-# Usa una imagen base de OpenJDK 17
-FROM openjdk:17-jdk-alpine
+# Etapa de construcción
```

Comment:
this is nos building correctly check https://github.com/ReBy298/FlightSearch/pull/3/files#diff-022779686499ed2a0c5b0a37c5d53a6f7b45627efaca9537b396459721c46f86

Reply:
Changed to FROM openjdk:17-jdk-alpine AS build